src/Components/Filter/Filter.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Filters } from '../../types/Filters';
+
+type Props = {
+  filter: Filters;
+  changeFilter: (value: Filters) => void;
+};
+
+const Filter: React.FC<Props> = ({ filter: selectedTodo, changeFilter }) => (
+  <nav className="filter">
+    <a
+      href="#/"
+      className={classNames(
+        'filter__link',
+        { selected: selectedTodo === Filters.All },
+      )}
+      onClick={() => changeFilter(Filters.All)}
+    >
+      All
+    </a>
+
+    <a
+      href="#/active"
+      className={classNames(
+        'filter__link',
+        { selected: selectedTodo === Filters.ACTIVE },
+      )}
+      onClick={() => changeFilter(Filters.ACTIVE)}
+    >
+      Active
+    </a>
+
+    <a
+      href="#/completed"
+      className={classNames(
+        'filter__link',
+        { selected: selectedTodo === Filters.COMPLETED },
+      )}
+      onClick={() => changeFilter(Filters.COMPLETED)}
+    >
+      Completed
+    </a>
```

Comment:
try not repeat yourself.
For these links you can create an array with objects (outside the component) and then map this array

Example:
```
const myLinks = [
{id: 1, href: '/', type: Filters.ACTIVE},
]

<ul>
{myLinks.map(link => (
  <li {{your_item_values}}>
    <a {{your_link_values}}>
    ...
    </a>
  </li>
))}
</ul>
```